### ISSUE TITLE
Failed to collect stats on running processes

### ISSUE THREAD

**Issue by pranavmalaviya** [tidalmigrations/machine_stats#149]
Getting `too many values to unpack` error when running `machine-stats --process-stats`.

```
$ machine-stats --process-stats
ERROR! ubuntu@ec2-13-55-68-62.ap-southeast-2.compute.amazonaws.com: too many values to unpack (expected 2)
{
    "servers": [
        {
            "cpu_count": 1,
            "cpu_name": "Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz",
            "custom_fields": {
                "cpu_average": 0.28035695380726994,
                "cpu_peak": 2.970297029702973,
                "cpu_sampling_timeout": 30,
                "cpu_utilization_timestamp": "2022-01-18 12:58:56"
            },
            "fqdn": "ip-172-31-46-70.ap-southeast-2.compute.internal",
            "host_name": "ip-172-31-46-70",
            "ip_addresses": [
                "172.17.0.1",
                "172.31.46.70",
                "fe80::4b9:dcff:fe21:390"
            ],
            "operating_system": "Ubuntu",
            "operating_system_version": "18.04",
            "ram_allocated_gb": 0.951171875,
            "ram_used_gb": 0.689453125,
            "storage_allocated_gb": 7.908359527587891,
            "storage_used_gb": 2.914031982421875
        }
    ]
}
MACHINE STATS RECAP
ubuntu@ec2-13-55-68-62.ap-southeast-2.compute.amazonaws.com : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

### Steps to Reproduce
1. Create an AWS EC2 machine and add its configs to `hosts` file
2. Run `machine-stats --process-stats`

**Comment by SamDesmondKing** [comment]
This is fixed by this PR #189 , maybe we can close the issue. WDYT @jmariomejiap ?

**Comment by jmariomejiap** [comment]
> This is fixed by this PR #189 , maybe we can close the issue. WDYT @jmariomejiap ?

Yes, Good point. lets close it
